test/test_runner.rb

```diff
@@ -28,22 +28,6 @@ def run_and_assert_error_messages(argv, expects)
     assert_equal(1, $CHILD_STATUS.exitstatus)
   end
 
-  def test_local_mode
-    report_diff = ReportDiff.new
-    PmdReportBuilder.any_instance.stubs(:build).returns(nil).twice
-    DiffBuilder.any_instance.stubs(:build).returns(report_diff).twice
-
-    argv = %w[-r target/repositories/pmd -b master -bc config/design.xml
-              -p pmd_releases/6.1.0 -pc config/design.xml -l test/resources/project-test.xml]
-    Runner.new(argv).run
-
-    actual_file_content = File.open('target/reports/diff/checkstyle/index.html').read
-    expected_file_content =
-      File.open('test/resources/html_report_builder/expected_empty_diff_report.html').read
-
-    assert_equal(expected_file_content, actual_file_content)
-  end
-
```

Comment:
This test case actually test the `HtmlReportBuilder` rather than `Runner`, so I remove it.

Reply:
:+1: Yes, it's this test case: https://github.com/pmd/pmd-regression-tester/blob/568ed1c42f3acfc0468a1756a7fe115c28a56254/test/test_html_report_builder.rb#L60

Reply:
Actually, thinking this over: If we remove this test case, then we don't have a test, anymore, that runs the "happy path" - we are left with only error cases. We probably don't need to assert here the actual created files, but the important part is the interaction between Runner, PmdReportBuilder and DiffBuilder. And shouldn't there also be a HtmlBuilder involved?

Reply:
> but the important part is the interaction between Runner, PmdReportBuilder and DiffBuilder.

This make sence, I'll re-add the test case.